spec/requests/admin/users_manage_spec.rb

```diff
@@ -164,6 +164,39 @@ def full_profile
       put "/admin/users/#{user.id}", params: { user: { remove_credits: "3" } }
       expect(user.credits.size).to eq(2)
     end
+
+    it "removes non-admin roles from non-super_admin users" do
```

Comment:
In examples that contain more than one assertion we tend to [aggregate failures](https://relishapp.com/rspec/rspec-core/docs/expectation-framework-integration/aggregating-failures#use-%60:aggregate-failures%60-metadata) by using RSpec [metadata](https://relishapp.com/rspec/rspec-core/docs/expectation-framework-integration/aggregating-failures#use-%60:aggregate-failures%60-metadata):

```suggestion
    it "removes non-admin roles from non-super_admin users", :aggregate_failures do
```

Reply:
Great call!